inscriptions/ISic1654.xml

```diff
@@ -171,18 +176,31 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
+                <ab>
+                    <lb n="1"/><g type="cross">☩</g> Hic requiescit in pa<supplied reason="lost">ce</supplied>
+                    <lb n="2"/><expan><abbr>b</abbr><ex>o</ex><abbr>n</abbr><ex>ae</ex></expan> <expan><abbr>m</abbr><ex>emoriae</ex></expan> Busiridis su<supplied reason="lost">b</supplied> <supplied reason="lost">die</supplied>
+                    <lb n="3"/><num value="17"><supplied reason="lost">X</supplied>VII</num> <expan><abbr>kal</abbr><ex>endas</ex></expan> septemb<supplied reason="lost">res</supplied>
+                    <lb n="4"/><expan><abbr><supplied reason="lost">i</supplied>nd</abbr><ex>ictione</ex></expan> septima con<supplied reason="lost">sulatu</supplied> <supplied reason="lost">Probi</supplied>
```

Comment:
Tag the septima as a number?

Reply:
I also wonder if we should split the two epitaphs in two separate `<ab>`s, to flag that they are two different texts about two different people.

Reply:
I am not sure about that... It depends on whether this can be useful for future work